hasher-matcher-actioner/webapp/src/App.jsx

```diff
@@ -3,16 +3,15 @@
  */
 
 import React from 'react';
-import {BrowserRouter as Router, Route} from 'react-router-dom';
-import {AnimatedSwitch} from 'react-router-transition';
```

Comment:
Definitely fine to remove but it looks these change this broke/removed the padding/grid on some of the subpages

Reply:
That's intentional. I want sub-pages to have the capability to use real estate as they see fit. We can provide common layouts as compose helpers.

eg. Matches page needs to use _all_ the available space. Settings probably does not.

Reply:
Okay, someone should own updating this soon for all subpages but not blocking.